__init__.py

```diff
@@ -84,6 +85,7 @@ def currency_filter(num, grouping=True):
     return locale.currency(num, grouping=grouping)
 
 if __name__ == '__main__':
+    nr_agent.initialize()
```

Comment:
Good catch, @jmcarp . Yes, let's standardize on the wrapper script and take this import/initialize out.